### ISSUE TITLE
TODO: officially release this action

### ISSUE THREAD

**Issue by DilumAluthge** [JuliaRegistries/compathelper-action#21]
This action has not actually been officially released yet, because I want to do some more testing.

**Comment by gerlero** [comment]
@DilumAluthge Are there still plans to release this action?

Makes a lot of sense to me to have the CompatHelper “boilerplate” defined just once in an officially sanctioned action…

**Comment by DilumAluthge** [comment]
Yeah, I would like to eventually release this, but it needs more testing first.
